alarm-frontend/src/Components/AddAlarm.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { minutesNumber, hourNumber } from "../fixNumbers";
+import useOption from "../Hooks/useOption";
+import axios from "axios";
+
+const BASE_URL = "http://127.0.0.1:5000";
+
+function AddAlarm({ handler }) {
+    const [hour, setHour] = useOption("Hour");
+    const [minutes, setMinutes] = useOption("Minutes");
+    const [amPmOption, setAmPmOption] = useOption("Am-Pm");
+    const [type, setType] = useOption("type");
+    const [difficulty, setDiff] = useOption("diff");
+
+    async function newAlarm() {
+        if (
+            hour !== "Hour" &&
+            minutes !== "Minutes" &&
+            amPmOption !== 'Am-Pm' &&
+            type !== "type" &&
+            difficulty !== "diff"
+        ) {
+            await axios.post(`${BASE_URL}/alarms/add`, {
+                hour, minutes, amPmOption, type, difficulty
+            })
+            handler();
+        } else {
+            alert("Please select an option for each field")
+        }
+    }
+
+    return (
+        <div>
+            <div>
+                <select {...setHour}>
+                    <option disabled value="Hour">
```

Comment:
should use a label instead of a disabled option
<label for="cars">Choose a car:</label>